src/dft/backends/mklcpu/mkl_dft_cpu_wrappers.cpp

```diff
@@ -21,11 +21,9 @@
 #include "dft/function_table.hpp"
 
 #define WRAPPER_VERSION 1
+#define BACKEND         mklcpu
 
 extern "C" dft_function_table_t mkl_dft_table = {
     WRAPPER_VERSION,
-    oneapi::mkl::dft::mklcpu::create_commit,
-    oneapi::mkl::dft::mklcpu::create_commit,
-    oneapi::mkl::dft::mklcpu::create_commit,
-    oneapi::mkl::dft::mklcpu::create_commit
+#include "dft/backends/backend_wrappers.cxx"
```

Comment:
There is some black magic going on here. Can't understand what this table is for. Some extra clarifying comments might be helpful (either here or on dft_function_table_t). Maybe I'm just not familiar enough with oneMKL

Reply:
I've added a comment in [d892e17](https://github.com/hjabird/oneMKL/pull/4/commits/d892e177b3bd2f559fbde3cc4b2684008abfac2d).

This is for filling in the function table that each backend gives to the main library for dispatch at runtime. This function table is filled with the backend-specific implementations like `mklgpu::compute_forward<types...>`. This should be the same for every backend, except replacing `mklgpu` with whatever the backend's name is.

This technique is also used in other domains. For example see `src/blas/backend_wrappers.cxx`. For the BLAS version, the file is nicer since template paramaters can be deduced, but for the DFT version explicit values are required.